README.md

```diff
@@ -23,20 +21,55 @@ See [examples](examples) folder.
 var EddystoneBeaconScanner = require('eddystone-beacon-scanner');
 ```
 
-### Register discover event handler
+### Register event handlers
 
+##### discover
 ```javascript
 EddystoneBeaconScanner.on('discover', function(beacon) {
   // ...
 });
 ```
 
-The ```beacon``` object will have the following properties:
+##### found
+```javascript
+EddystoneBeaconScanner.on('found', function(beacon) {
+  // ...
+});
+```
+
+##### lost
+
+```javascript
+EddystoneBeaconScanner.on('lost', function(beacon) {
+  // ...
+});
+```
+
+The ```beacon``` object will have the following properties depending on the frame type:
+##### URL
 
+ * ```type``` - Eddystone type
+ * ```txPower``` - Measured received power at 0 m in dBm
  * ```url``` - (expanded) URL the beacon is broadcasting
- * ```type``` - 'url'
- * ```txPower``` - measured received power at 0 m in dBm
  * ```rssi``` - RSSI of discovered beacon
+ * ```distance``` - Approximate distance from beacon
+
+##### UID
+ * ```type``` - Eddystone type
+ * ```txPower``` - Measured received power at 0 m in dBm
+ * ```namespace``` - 10-byte ID Namespace
+ * ```instance``` - 6-byte ID Instance
+ * ```rssi``` - RSSI of discovered beacon
+ * ```distance``` - Approximate distance from beacon
+
+##### TLM
```

Comment:
Cool, I've split it out and also updated the read me.

@jacobrosenthal any feedback on the updated read me?